src/features/projectsV2/ProjectDetails/components/OtherInterventionInfo.tsx

```
@@ -0,0 +1,140 @@
+    return { totalTreesCount, plantedLocationArea };
+  }, [plantLocationInfo?.geometry, plantLocationInfo?.type]);
+
+  const sampleInterventionSpeciesImages = useMemo(() => {
+    if (plantLocationInfo && plantLocationInfo.sampleInterventions.length>0) {
+      const result = plantLocationInfo.sampleInterventions && plantLocationInfo.sampleInterventions.map((item) => {
+        return {
+          id: item.coordinates[0].id,
+          image: item.coordinates[0].image ?? '',
+          description: tProjectDetails('sampleTreeTag', { tag: item.tag }),
+        };
+      });
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)